### PR TITLE
Onboarding scripts, user and staff mapping load scripts and instructions.

### PR DESCRIPTION
**Pull request by timharrison-moj**
These were the scripts used to load the North East user data and will be useful for other regions to follow.

### CODE REVIEW

File: data-load/user-load.sh [hunk 1]
```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+#
+# Script to load a batch of new users into OAuth for new service access.
+#
+# Parameters:
+#       1. ENV - [t3|t2|preprod|prod]
+#       2. CLIENT - the client Id and secret, comma-separated
```
Comment (petergphillips) [inline data-load/user-load.sh]: should be colon separated I think

File: data-load/user-load.sh [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  HOST="https://gateway.$ENV.nomis-api.service.hmpps.dsd.io"
+fi
+
+# Check whether the file exists and is readable
+if [[ ! -f "$FILE" ]]; then
+  echo "Unable to find file $FILE"
+  exit 1
+fi
+
+# Get  token for the client name / secret and store it in a hidden file .access_token
```
Comment (petergphillips) [inline data-load/user-load.sh]: doesn't store in a hidden fie, we just use `jq` to extract out the `access_token` from the json

File: data-load/user-load.sh [hunk 3]
```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+#
+# Script to load a batch of new users into OAuth for new service access.
+#
+# Parameters:
+#       1. ENV - [t3|t2|preprod|prod]
```
Comment (petergphillips) [inline data-load/user-load.sh]: won't work for t2 out of the box as the `HOST` will be incorrect

File: data-load/user-load.sh (hunk 1 above)
Comment (timharrison-moj) [reply data-load/user-load.sh]: Good spot. Wrong words though does the right thing.

File: data-load/user-load.sh (hunk 2 above)
Comment (timharrison-moj) [reply data-load/user-load.sh]: Corrected to stored in env var TOKEN.

File: data-load/user-load.sh (hunk 3 above)
Comment (timharrison-moj) [reply data-load/user-load.sh]: Ok - I added the correct gateway address for using t2 so it would work there too.

**Pull request merged**